specs/RestartRequested.md

```diff
@@ -98,10 +175,19 @@ s
 ```c#
 namespace Microsoft.Web.WebView2.Core
 {
+    enum CoreWebView2RestartPriorityLevel
+    {
+        BestEffort = 0,
+        Critical = 1,
+    };
+
     runtimeclass CoreWebView2Environment
     {
         // ...
         event Windows.Foundation.TypedEventHandler<CoreWebView2Environment, Object> RestartRequested;
+
+        // Set restart priority level
+        CoreWebView2RestartPriorityLevel RestartPriorityLevel { get; set; };
```

Comment:
I thought this was a get only property on the RestartRequested event args?

Reply:
Synced offline. Will make the change